data/certChooser.js

```diff
@@ -66,4 +65,6 @@ document.addEventListener("keydown", keyboardListener);
 document.addEventListener("focus", function(event) {
   document.getElementById("certPassword").focus();
   event.target.removeEventListener(event.type, arguments.callee);
+  var headline = document.getElementById("headline");
+  headline.textContent = headline.textContent.replace("<site here>", data.domain);
```

Comment:
Probably you are right. Headline text change did not worked on it's original place, so I moved it to focus event handler.